### ISSUE TITLE
rrd permission denied.

### ISSUE THREAD

**Issue by lindberg87** [motoz/PellMon#111]
Hey Pellmon.

I got a error then Pellmon starts up, I have take a screenshot from the terminal you can see on this link -> [https://photos.app.goo.gl/8ncrof6JJxW9mcYw9]

What can I do about it ? 

Best Regards Stig

**Comment by omakood** [comment]
I got similar error from this morning:
`2020-12-30 05:40:22,045 - INFO - rrdtool update 1609299621:0.0:25.2:22.0:0.0:0.0:20.7:0.0:0.0:0.0:99.0:U:U:U:U:U:94649:990.0:1609289097 failed with, , ERROR: could not lock RRD `

**Comment by motoz** [comment]
Can't be sure from the screenshot, but it looks like you are trying to run pellmonsrv as the 'pi' user with a normal setup where the database is owned by the 'pellmonsrv' system user. The 'pi' user isn't allowed to write to the database in a standard installation..

**Comment by motoz** [comment]
It would help if you would explain a bit what you are trying to do, I get the feeling that you are doing something more than just trying to install pellmon. Are you doing some development?

**Comment by lindberg87** [comment]
> pellmonsrv

Yes I run the raspian as pi user. Should I instead run as pellmonsrv user ? and what are the password for this account ? 
I think that could be the solution for my problem. 

Best regards.

**Comment by motoz** [comment]
I still don't know why you are trying to run pellmonsrv in this way. If you have changed something and want to run it it in the foreground (as opposed to daemon mode), then I suggest you follow this guide: https://github.com/motoz/PellMon/wiki/Contributing-to-PellMon . If you want to run the already installed pellmonsrv in foreground, then you can look in the initscript to see how it's started in /etc/init.d/pellmonsrv (I don't remember all details). In both cases you need to first stop the already running instance if you have it installed.

There is no password for the system user pellmonsrv. You could use 'su' to run the command as pellmonsrv, but it's actually designed to start as root and drop the privileges later (when running as installed). But if you follow the 'contributing' guide all details are taken care of and you don't need to mess with the system installation.
